src/views/InProgress.vue

```diff
@@ -395,19 +403,28 @@ export default defineComponent({
                 const resp = await OrderService.packOrders({
                   shipmentIds
                 });
-                if (resp.status === 200 && !hasError(resp)) {
-                  showToast(translate('Orders packed successfully'));
-                } else {
+                if (hasError(resp)) {
```

Comment:
When having error we direct the flow to catch block in that case the loader will not be dismissed.

Reply:
Updated sir, handled in finally block